content/docs/user-guide/assets/asset-processor/skip-startup-scan.md

```diff
@@ -0,0 +1,36 @@
+---
+linkTitle: Skip Startup Scan
+title: Asset Processor Skip Startup Scan
+description: Skips startup checks for assets modified while Asset Processor was closed.
+weight: 700
+toc: true
+---
+
+Skipping Startup Scan improves O3DE Editor's launch time if no assets are modified while Asset Processor was closed. You can switch startup scan modes at any time but need to restart Asset Processor for the change to take effect. Asset Processor saves your preferences between sessions.
+
+Skipping Startup Scan is an expert mode that is designed for aiding code development workflows (e.g. code updating and debugging). Use it only when you are not modifying the asset processing pipeline or making asset content changes.
+
+When Skip Startup Scan is enabled:
+1. Asset Processor skips startup checks on asset modifications after launch and enters the idle state directly.
+2. O3DE Editor receives a message from Asset Processor which indicates that all the assets are ready to use.
+3. O3DE Editor is unblocked to launch.

```

Comment:
nit: Maybe add a line that AP will begin to watch for asset changes once it goes to Idle state?

Reply:
Just added the line.